app/lib/support_ticket.rb

```diff
@@ -0,0 +1,58 @@
+require "gds_zendesk/client"
+require "gds_zendesk/dummy_client"
+
+class SupportTicket
```

Comment:
I don't think this really models a SupportTicket - Should we call it a SupportTicketCreator? Since it's job seems to be to do the transaction of creation rather than to model a domain concept.